py/server/deephaven/time.py

```diff
@@ -85,21 +81,16 @@ def today(tz: TimeZone) -> str:
         DHError
     """
     try:
+        if tz is None:
+            tz = _JDateTimeUtils.timeZone()
+
         return _JDateTimeUtils.today(tz)
     except Exception as e:
         raise DHError(e) from e
 
 
-# endregion
-
-# region Time Zone
-
-
-def time_zone(tz: Optional[str]) -> TimeZone:
-    """ Gets the time zone for a time zone name.
-
-    Args:
-        tz (Optional[str]): Time zone name.  If None is provided, the system default time zone is returned.
+def dh_time_zone() -> TimeZone:
```

Comment:
We have a `get_server_timezone()` in `config/_init__.py`, should we remove it?

Reply:
I'm not sure.

Reply:
I'm probably not opposed to removing it.

Reply:
let's do nothing for now.